### ISSUE TITLE
Accessibility issue on your site.

### ISSUE THREAD

**Issue by YashuMishra** [FossifyOrg/fossifyorg.github.io#4]
Few common Accessibility issues which i came around while auditing your URL : https://hosted.weblate.org/projects/fossify

1.Luminosity ratio of the navigation controls such as 'Languages', 'Info', 'Search', 'Insight'..etc are less than expected 4.5:1 which violates Wcag 1.4.3.

2. Alternate text is not defined for 'Translated 45%' img

**Comment by Aga-C** [comment]
It's not our website, it's an external system for doing translations which we are using. You can raise this issue on their repository: https://github.com/WeblateOrg/weblate/.
